lte/gateway/c/core/oai/test/amf/test_amf.cpp

```diff
@@ -10,11 +10,15 @@
  */
 
 #include "util_nas5g_pkt.h"
-#include "dynamic_memory_check.h"
 #include <gtest/gtest.h>
+#include "intertask_interface.h"
```

Comment:
is this header used? If not remove it.

Reply:
grpc_service_task_zmq_ctx was declared in intertask_interface.h